src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Snapshot/Dependency.cs

```diff
@@ -266,7 +266,9 @@ private static string GetAlias(IDependency dependency)
 
         private static string Normalize(string id)
         {
-            return id.Replace('/', '\\');
+            return id
+                .Replace('/', '\\')
+                .Replace("..", "__");
```

Comment:
This could be problematic. This ID becomes the canonical name of the `IVsHierarchy` item, and there are places we expect to be able to extract the original file path from the canonical name (for example, analyzer support in Solution Explorer).